### PR TITLE
Bug 1498700 - upgrade all services, notably hooks for tc-lib-pulse

### PR DESCRIPTION
**Pull request by djmitche**
/cc @alexandrasp

### CODE REVIEW

**Comment by djmitche** [comment]
OK, with typo fixed, that works a lot better.

**Comment by alexandrasp** [comment]
nice! :)

File: hooks-service.tf [hunk 1]
```diff
@@ -33,7 +33,7 @@ module "hooks_secrets" {
     MONITORING_ENABLE        = "false"
     PUBLISH_METADATA         = "false"
     AZURE_ACCOUNT            = "${azurerm_storage_account.base.name}"
-    PULSE_USERNAME           = "${module.events_rabbitmq_user.username}"
+    PULSE_USERNAME           = "${module.hooks_rabbitmq_user.username}"
```
Comment (imbstack) [inline hooks-service.tf:36]: Hmm, we keep doing this. There must be some way to clean this up and make it easy.

/me makes a note to try at some point.

**Review by imbstack — APPROVED** [review]

File: hooks-service.tf (hunk 1 above)
Comment (djmitche) [reply hooks-service.tf:36]: I suspect it was one episode of copy-pasta.  Once all of the services are running, it should be fine (and this is the last one).

**Pull request merged**